### PR TITLE
LPS-88873 Upgrade ASM to 7.0

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian This PR upgrades ASM in lib/portal to 7.0, and also updates the several modules that have compile dependencies of ASM. portal-template-soy-impl has an included dependency of asm-commons 6.0 which seems to be used by Google Closure Compiler, so we didn't touch it.

### CODE REVIEW

**Comment by dantewang** [comment]
Previous ci-test: https://github.com/dantewang/liferay-portal/pull/956#issuecomment-452215054
I can see the DDM tests randomly fail on various PRs. Xiangyue ran them locally on top of this PR and they passed.

**Comment by tinatian** [comment]
@dantewang @caixiangyue 
Since asm-debug-all has been separated into different jars, I am wondering if we really need all of them, could you please check the usages and only include the ones we really depend on.

Thanks.
Tina.

File: modules/apps/static/portal-configuration/portal-configuration-metatype-api/build.gradle [hunk 1]
```diff
@@ -4,7 +4,11 @@ dependencies {
 	compileOnly group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.metatype", version: "1.3.0"
-	compileOnly group: "org.ow2.asm", name: "asm-debug-all", version: "5.0.4"
+	compileOnly group: "org.ow2.asm", name: "asm", version: "7.0"
```
Comment (tinatian) [inline modules/apps/static/portal-configuration/portal-configuration-metatype-api/build.gradle:7]: @dantewang 
You changed three build.gradle files, could you please check if we need to include every jar in each of them?

This should have nothing to do with the export package from system.packages.extra.bnd.

Thanks.
Tina.

**Comment by dantewang** [comment]
@tinatian sure

Tina Tian <notifications@github.com> 于 2019年1月9日周三 08:59写道：

**Pull request closed without merging**